stock_picking_partner_note/models/stock_picking.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2024 Camptocamp (<https://www.camptocamp.com>).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+
+from odoo import api, fields, models
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    note = fields.Html(compute="_compute_note", store=True)
```

Comment:
as this is not a "regular" note, I would've named it "preparation_note" or smth like that.